data/lib/core/game.lua

```diff
@@ -189,6 +189,12 @@ function Game.getUnpromotedVocations()
 end
 
 do
+	-- World Light
+	-- NOTE: if defaultWorldLight is set to true the world light algorithm will
+	-- be handled in the sources. set it to false to avoid conflicts if you wish
+	-- to make use of the function Game.setWorldLight(color, level)
```

Comment:
Is this comment still true? Is it handled in the sources or is it handled in Lua? `data/scripts/globalevents/world_light.lua` seems to be doing just that.